plugins/attachment-resources/src/components/icons/FileDownload.svelte

```diff
@@ -0,0 +1,28 @@
+<!--
+// Copyright © 2022 Anticrm Platform Contributors.
```

Comment:
It's extra, use only `// Copyright © 2022 Hardcore Engineering Inc.`

Reply:
Done